src/main/scala/com/microsoft/hyperspace/index/IndexLogEntry.scala

```diff
@@ -542,6 +543,10 @@ case class IndexLogEntry(
         IndexConstants.HAS_PARQUET_AS_SOURCE_FORMAT_PROPERTY, "false").toBoolean
   }
 
+  def logVersion: String = {
+    properties.getOrElse(IndexConstants.INDEX_LOG_VERSION_PROPERTY, "undefined")
+  }
```

Comment:
instead of introducing a key in the map, can't we simply use super.id directly here?
i mean 
```
def logVersion: String = id.toString
```

If we do this, we will not even need to introduce `endId()` def in Actions.scala class
Action.begin(),
Action.end() set this id themselves and it's always up to date with the actual file name of the log entry file.

cc @imback82

Reply:
Thanks! I updated this PR to use LogEntry.id. 

Btw I added `endId` in #272 as the id is required in `op()`.. @apoorvedave1 Could you also have a look at the change?